configs/config.dell_optiplex_9010_sff_uefi_txt

```diff
@@ -0,0 +1,53 @@
+CONFIG_USE_OPTION_TABLE=y
+CONFIG_USE_BLOBS=y
+CONFIG_VENDOR_DELL=y
+CONFIG_BOARD_DELL_OPTIPLEX_9010=y
+CONFIG_ONBOARD_VGA_IS_PRIMARY=y
+# CONFIG_DRIVERS_UART_8250IO is not set
+CONFIG_PCIEXP_CLK_PM=y
+CONFIG_HAVE_EM100_SUPPORT=y
+CONFIG_LOCALVERSION="v0.1.0"
+CONFIG_PAYLOAD_EDK2=y
+CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/dasharo/evaluation_logo.bmp"
+CONFIG_EDK2_BOOT_MENU_KEY=0x0011
+CONFIG_EDK2_GOP_DRIVER=y
+CONFIG_EDK2_REPOSITORY="https://github.com/Dasharo/edk2.git"
+CONFIG_EDK2_TAG_OR_REV="origin/rebased"
+CONFIG_EDK2_SETUP_MENU_KEY=0x000C
+# CONFIG_EDK2_CBMEM_LOGGING=y
+CONFIG_EDK2_FOLLOW_BGRT_SPEC=y
+CONFIG_EDK2_SD_MMC_TIMEOUT=1000
+CONFIG_TPM_MEASURED_BOOT=y
+CONFIG_VENDOR_DELL=y
+CONFIG_BOOTSPLASH_FILE="dasharo-black.bmp"
+CONFIG_BOOTSPLASH_IMAGE=y
+CONFIG_GENERIC_LINEAR_FRAMEBUFFER=y
+CONFIG_LINEAR_FRAMEBUFFER=y
+CONFIG_TPM2=n
+CONFIG_EDK2_SERIAL_SUPPORT=y
+CONFIG_EDK2_DASHARO_SERIAL_REDIRECTION_DEFAULT_ENABLE=y
+CONFIG_BUILD_IPXE=y
+CONFIG_IPXE_ADD_SCRIPT=y
+CONFIG_IPXE_CUSTOM_BUILD_ID="12345678"
+CONFIG_IPXE_NO_PROMPT=y
+CONFIG_EDK2_ENABLE_IPXE=y
+CONFIG_EDK2_IPXE_OPTION_NAME="Network Boot and Utilities"
+CONFIG_EDK2_DASHARO_SYSTEM_FEATURES=y
+CONFIG_EDK2_DASHARO_SECURITY_OPTIONS=y
+CONFIG_EDK2_DASHARO_USB_CONFIG=y
+CONFIG_EDK2_DASHARO_CHIPSET_CONFIG=y
+CONFIG_EDK2_DASHARO_NETWORK_BOOT_DEFAULT_ENABLE=y
+CONFIG_EDK2_PLATFORMS_REPOSITORY="https://github.com/Dasharo/edk2-platforms"
+CONFIG_EDK2_USE_EDK2_PLATFORMS=y
+CONFIG_EDK2_PLATFORMS_TAG_OR_REV="3323ed481d35096fb6a7eae7b49f35eff00f86cf"
+CONFIG_SMSC_SCH5545_EC_FW_FILE="sch5545_ecfw.bin"
+CONFIG_INCLUDE_SMSC_SCH5545_EC_FW=y
+CONFIG_EDK2_SETUP_PASSWORD=y
+CONFIG_EDK2_DASHARO_NETWORK_CONFIG=y
+CONFIG_INTEL_TXT=y
+CONFIG_INTEL_TXT_BDR_VERSION=4
+CONFIG_INTEL_TXT_BIOSACM_FILE="IVB_BIOSAC_PRODUCTION.BIN"
+CONFIG_INTEL_TXT_SINITACM_FILE="SNB_IVB_SINIT_20190708_PW.bin"
```

Comment:
That's an old file. You should take one from this link: https://cdrdv2.intel.com/v1/dl/getContent/630744

Reply:
@miczyg1 if I understand correctly, I have to take a `SNB_IVB_`-prefixed file, there's only one such file in the zip you've linked, and that's precisely the one I'm using in the config:

```
flewinski in ~/Downloads/630744_003(1)/630744_003 λ ls
 630744_AcmErrorsDoc_003.csv
 630744_ACM_Platform_List_003.pdf
 ADL_SINIT_v1_18_16_20230427_REL_NT_O1.PW_signed.bin
 BDW_SINIT_20190708_1.3.2_PW.bin
 CFL_SINIT_20221220_PRODUCTION_REL_NT_O1_1.10.1_signed.bin
 CML_RKL_S_SINIT_v1.13.33_REL_NT_O1.PW_signed.bin
 CML_S_SINIT_1_13_33_REL_NT_O1.PW_signed.bin
 CMLSTGP_SINIT_v1_14_46_20220819_REL_NT_O1.PW_signed.bin
 desktop.ini
'Legal Disclaimer.pdf'
 RKLS_SINIT_v1_14_46_20220819_REL_NT_O1.PW_signed.bin
 SKL_KBL_AML_SINIT_20211019_PRODUCTION_REL_NT_O1_1.10.0.bin
 SNB_IVB_SINIT_20190708_PW.bin
 TGL_SINIT_v1_14_46_20220819_REL_NT_O1.PW_signed.bin
```

Reply:
@filipleple my bad, I though SNB/IVB has an SINIT ACM from 2021, but I think I have mistaken it with SKL_KBL_AML_SINIT